example-KG-COVID-19-usage.ipynb

```diff
@@ -0,0 +1,577 @@
+    "    ignore_duplicated_edges=True,\n",
+    "    ignore_duplicated_nodes=True,\n",
+    ");\n",
+    "\n",
+    "training.report()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'nodes_number': '377577',\n",
+       " 'is_directed': 'false',\n",
+       " 'density': '0.00021710748243868716',\n",
+       " 'degrees_min': '0',\n",
+       " 'traps_rate': '0.022019349695558788',\n",
+       " 'bidirectional_rate': '1',\n",
+       " 'degrees_mean': '81.97479189675218',\n",
+       " 'selfloops_rate': '0.00001554029368764255',\n",
+       " 'connected_components_number': '9068',\n",
+       " 'degrees_mode': '1',\n",
+       " 'singleton_nodes': '8314',\n",
+       " 'degrees_max': '90378',\n",
+       " 'edges_number': '30951796',\n",
+       " 'unique_node_types_number': '37',\n",
+       " 'unique_edge_types_number': '33',\n",
+       " 'strongly_connected_components_number': '9068',\n",
+       " 'degrees_median': '6'}"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "graph = EnsmallenGraph.from_csv(\n",
+    "    edge_path=\"merged-kg_edges.tsv\",\n",
+    "    sources_column='subject',\n",
+    "    destinations_column='object',\n",
+    "    directed=False,\n",
+    "    edge_types_column='edge_label',\n",
+    "    default_edge_type='biolink:Association',\n",
+    "    node_path=\"merged-kg_nodes.tsv\",\n",
+    "    nodes_column='id',\n",
+    "    default_node_type='biolink:NamedThing',\n",
+    "    node_types_column='category',\n",
+    "    ignore_duplicated_edges=True,\n",
+    "    ignore_duplicated_nodes=True,\n",
+    "    force_conversion_to_undirected=True # deprecated, removed in ensmallen_graph 0.4\n",
+    ");\n",
+    "graph.report()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### See [these](https://github.com/monarch-initiative/embiggen/blob/master/notebooks/) notebook to generate embeddings from the KG you've created above. There are notebooks to make embeddings using:\n",
+    "- [Skipgram](https://github.com/monarch-initiative/embiggen/blob/master/notebooks/Graph%20embedding%20using%20SkipGram.ipynb)\n",
+    "- [CBOW](https://github.com/monarch-initiative/embiggen/blob/master/notebooks/Graph%20embedding%20using%20CBOW.ipynb)\n",
+    "- [GloVe](https://github.com/monarch-initiative/embiggen/blob/master/notebooks/Graph%20embedding%20using%20GloVe.ipynb)\n",
+    "\n",
+    "#### These embeddings can then be used to train MLP, random forest, decision tree, and logistic regression classifiers using [this notebook](https://github.com/monarch-initiative/embiggen/blob/master/notebooks/Classical%20Link%20Prediction.ipynb).\n",
+    "\n",
+    "##### Note: consider running the code in these notebooks on a server with GPUs in order to complete in a reasonable amount of time"
```

Comment:
what's reasonable? Should we give some rough examples here?

Reply:
Added some rough estimates of runtime